src/Web/Microsoft.NET.Sdk.Web.Targets/Sdk.DefaultItems.targets

```diff
@@ -0,0 +1,39 @@
+<!--
+***********************************************************************************************
+Sdk.DefaultItems.targets
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your projects from the command-line or the IDE.
+
+Copyright (c) .NET Foundation. All rights reserved.
+***********************************************************************************************
+-->
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <Import Project="$(MSBuildThisFileDirectory)BundledVersions.props"
+		Condition="Exists('$(MSBuildThisFileDirectory)BundledVersions.props')" />
+
+  <Choose>
+    <When Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp' AND '$(_TargetFrameworkVersionWithoutV)' == '2.1'" >
```

Comment:
I'm doing a minimal implementation here for now. Once we start to patch 2.1 (which will require separate versions for self contained and portable apps) and when we branch for 2.2 (multiple versions depending on tfm), we will need to update the implementation here. Should I implement that right now or should I wait to add it until needed?

Reply:
Better to add it now to make sure it works IMO. Functional changes in patches are harder to get approved.

Reply:
There's very little that can be tested since right now the versions for portable and self contained apss are the same, but I'll add the logic anyway.